stratum/hal/lib/common/openconfig_converter.cc

```diff
@@ -410,10 +410,10 @@ ::util::StatusOr<Chassis> ComponentToChassis(
     case OPENCONFIGHERCULESPLATFORMPLATFORMTYPE_P4_SOFT_SWITCH:
       to.set_platform(PLT_P4_SOFT_SWITCH);
       break;
-    case OPENCONFIGHERCULESPLATFORMPLATFORMTYPE_BAREFOOT_TOFINO:
+    case OPENCONFIGHERCULESPLATFORMPLATFORMTYPE_GENERIC_BAREFOOT_TOFINO:
       to.set_platform(PLT_GENERIC_BAREFOOT_TOFINO);
       break;
-    case OPENCONFIGHERCULESPLATFORMPLATFORMTYPE_BAREFOOT_TOFINO2:
+    case OPENCONFIGHERCULESPLATFORMPLATFORMTYPE_GENERIC_BAREFOOT_TOFINO2:
```

Comment:
What about the reverse direction?  There's a TODO from Yi, but I don't have the context for it.